serverless/serverless
# Added Support for Cloud Components

## What did you implement

Added support for cloud component to work alongside the first component's version.



## How can we verify it



Try running `serverless deploy` in a directory containing:

- components v1
- components v2
- framework v1

They should all work with their respective experiences.

## Todos

<details>
<summary>Useful Scripts</summary>


- `npm run test:ci` --> Run all validation checks on proposed changes
- `npm run lint:updated` --> Lint all the updated files
- `npm run lint:fix` --> Automatically fix lint problems (if possible)
- `npm run prettier-check:updated` --> Check if updated files adhere to Prettier config
- `npm run prettify:updated` --> Prettify all the updated files

</details>


**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
